style.css

```diff
@@ -0,0 +1,458 @@
+  background-color: white;
+  color: #ee523c;
+  border: 4px solid #ee523c;
+  padding: 10px;
+  width: 150px;
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 10px;
+  background-image: url('./images/background-headlin.svg');
+  padding: 100px;
+  font-family: 'lato', sans-serif;
+}
+
+h2.first {
+  font-weight: 600;
+  font-size: 1.2rem;
+  color: #ee523c;
+}
+
+h1 {
+  color: #ee523c;
+  font-size: 2.8rem;
+  line-height: 3rem;
+  font-family: 'COCOGOOSE', sans-serif;
+  padding-bottom: 18px;
+}
+
+.headlin-description {
+  width: 50%;
+}
+
+h2.second {
+  color: #606060;
+  font-weight: 700;
+  font-family: 'COCOGOOSE', sans-serif;
+}
+
+.main-program {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 3rem;
+  background-image: url(./images/main.background.png);
+  background-color: #272a32;
+  color: #f7f7f7;
+  height: 90vh;
+}
+
+.cards {
+  display: flex;
+  align-items: center;
+  padding-top: 3rem;
+  gap: 10px;
+}
+
+.card {
+  height: 75%;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  box-sizing: border-box;
+}
+
+.card-1 {
+  height: 16rem;
+  display: grid;
+  grid-template-columns: none;
+  grid-template-rows: 4rem 5rem 6rem;
+  justify-items: center;
+  margin: 0;
+  width: 13rem;
+  padding: 10px;
+  background-color: rgba(255, 255, 255, 0.1);
+  box-sizing: border-box;
+  align-items: center;
+}
+
+.card-1:hover {
+  border: 1px solid #fff;
+  transition: 1s all;
+}
+
+.img-card img {
+  width: 50px;
+  height: 50px;
+  align-items: center;
+}
+
+.title-card {
+  color: #ee523c;
+  font-size: 1rem;
+}
+
+.description-card {
+  font-size: 15px;
+}
+
+.imagediv img {
+  width: 100px;
+  height: 100px;
+  position: relative;
+  left: 16px;
+  top: 27px;
+}
+
+.imagediv {
+  background-image: url(./images/pngegg.png);
+  background-repeat: round;
+  width: 65px;
+  height: 65px;
+}
+
+.partner {
+  min-height: 220px;
+  background-color: #474a51;
+}
+
+.Partner-logo {
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  height: 9rem;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  column-gap: 3rem;
+}
+
+.Partner-logo li {
+  text-decoration: none;
+  list-style: none;
+}
+
+.logo-about img {
+  width: 50%;
+  height: 50%;
+}
+
+.right img {
+  width: 80%;
+  height: 100%;
+}
+
+.left img {
+  width: 80%;
+  height: 100%;
+}
+
+.Partner-logo li img {
+  width: 5.5rem;
+}
+
+.partner h3 {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  font-size: 2rem;
+  color: rgb(191, 187, 182);
+}
+
+.toolbar {
+  display: none;
+}
+
+.maincardsec {
+  height: 98%;
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  padding: 0 10%;
+}
+
+.main-page-1 {
+  color: #f7f7f7;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  padding-top: 3rem;
+}
+
+.border-c {
+  color: black;
+  font-size: 30px;
+}
```

Comment:
- [ ] Great job redesigning this section. 👏 To top it off, kindly change the `font-weight` used so that it matches the design of the Behance template. 👍 

### your Design

![image](https://user-images.githubusercontent.com/77106962/205457496-6c3da594-5d5b-4821-a0bf-dedc675b0979.png)

### Behance Template

![image](https://user-images.githubusercontent.com/77106962/205457522-0660a173-9581-4929-b4aa-d5124c85d442.png)